analysers/analyser_merge_power_plant_FR.py

```diff
@@ -33,33 +33,34 @@ def __init__(self, config, logger = None):
             title = T_('Power plant not integrated, geocoded at municipality level'))
 
         self.init(
-            "https://opendata.reseaux-energies.fr/explore/dataset/registre-national-installation-production-stockage-electricite-agrege-311217",
+            "https://opendata.reseaux-energies.fr/explore/dataset/registre-national-installation-production-stockage-electricite-agrege",
             "Registre national des installations de production d'électricité et de stockage",
             CSV(Geocode_Addok_CSV(
                 SourceOpenDataSoft(
                     attribution="data.gouv.fr:RTE",
-                    url="https://opendata.reseaux-energies.fr/explore/dataset/registre-national-installation-production-stockage-electricite-agrege-311217"),
+                    url="https://opendata.reseaux-energies.fr/explore/dataset/registre-national-installation-production-stockage-electricite-agrege"),
                 columns='commune', citycode='codeINSEEcommune', logger=logger)),
             Load_XY("longitude", "latitude",
-                where = lambda res: res.get('puisMaxRac') and float(res["puisMaxRac"]) > 1000,
-                map = lambda res: dict(res, **{"_geom": [
-                    float(res["_geom"][0]) + (Stablehash.stablehash(str(res)) % 200 - 100) * 0.00001,
-                    float(res["_geom"][1]) + (Stablehash.stablehash(str(res)) % 212 - 106) * 0.00001] })),
+                where = lambda res: res.get('puisMaxRac') and float(res["puisMaxRac"]) > 250,
+                map = lambda res: dict(res, **{"_x": float(res["_x"]) + (Stablehash.stablehash(str(res)) % 200 - 100) * 0.00001, "_y": float(res["_y"]) + (Stablehash.stablehash(str(res)) % 212 - 106) * 0.00001}),
+                unique = ("codeEICResourceObject",)),
             Conflate(
                 select = Select(
                     types = ["ways", "relations"],
                     tags = {"power": "plant"}),
                 conflationDistance = 5000,
+                osmRef = "ref:EU:ENTSOE_EIC",
```

Comment:
Adding this line will raise osmose issue for each power plant already inside OSM without the ref. Are you sure to want ?
If there is lot of missing ref, is better to avoid it.

Reply:
EIC references are very important, currently causing harm at Europe scale.
https://github.com/energy-modelling-toolkit/hydro-power-database/issues/2#issuecomment-1195551791

It may be ok to have a different low-severity class for such issues but this change is done on purpose to encourage adding ref:EU:ENTSOE_EIC on every power=plant found in opendata.

Reply:
Les power=plant qui ne produissent pas d'électricité mais, par exemple, que de la chaleur, n'ont pas d'EIC.
Pareil pour ce qui n'est pas relié au réseau.
Et dans les données on a que ce qui fait plus de 36 kW, je ne sais pas si un EIC est attribué a ce qui fait moins mais, si il existe, il va être compliqué à trouver.

Reply:
Il faut rester dans le périmètre prévu par l'analyse :
* le fichier source ne contient que des installations de production d'électricité (éventuellement co-génération de chaleur par ailleurs, non décrite).
* Ce qui n'est pas relié au réseau n’apparaît pas dans le registre.
* Le seuil de prise en compte est fixé à 250 kW, on ne sera donc pas concerné par les agrégats < 36 kW.

Il n'y aura donc pas d'alerte concernée par ces points.

Reply:
> L'un de nous 2 n'a pas compris correctement ce que Frodrigo a dit. Moi ce que je comprends, c'est qu'avec cette ligne osmose va remonter un problème pour tous les power=plant sans ref:EU:ENTSOE_EIC quoi qu'en dise le fichier source. Donc beaucoup de problèmes pour Osmose qui n'en sont pas.

Nope. As it is a `missing_official` analyser, It will only point OSM data without `ref:EU:ENTSOE_EIC` from the OpenData source.

Reply:
ok j'ai toujours pas compris.
"Adding this line will raise osmose issue for each power plant already inside OSM without the ref."
"It will only point OSM data without ref:EU:ENTSOE_EIC from the OpenData source."
Pour Osmose, il y aura un problème quand dans osm il y aura un ref:EU:ENTSOE_EIC non présent dans la source ?
Si c'est ça est ce que ça compte si c'est en dehors de la France ou sur d'autres objet que des power=plant ?

Reply:
Je pense que ce sont de bonnes questions, à part pour les objets autres que power=plant puisqu'il se trouve dans le static1.

Vu que le fichier opendata couvre seulement certains codes EIC, il faudrait faire une inclusion du fichier dans OSM et pas l'inverse.